README.md

```diff
@@ -5,15 +5,16 @@ commands:
     \list
     \regex
     \get
+    \help
 
 requirements:
 
     pip install --user irc
 
 usage:
 
-    python bot.py --server irc.server.tld --chan '#channel' --root /path/to/files/
+    python bot.py --server irc.server.tld --chan '#channel' --root /path/to/files/ --port 3000 --addr 178.23.78.23
```

Comment:
please use an ip in `192.0.2.0/24` as per rfc 3330 and mark this as a placeholder value.